generator/index.template.html

```diff
@@ -98,10 +98,12 @@ <h1>Usability Hub</h1>
       <!-- {{item.id}} -->
       <div class="item">
         {% for category in item.categories %}
+        {% if category.code_type == 'model' %}
```

Comment:
the syntax looks fine, however if we're only interested in categories of type model, we could filter that in the generator.py instead (the more logic we add in the html, the less readable it will become)

Reply:
Yeah. But I'm not sure how it will become after the users feedback. I'll keep it here for the moment.